openstates/de/committees.py

```diff
@@ -1,52 +1,54 @@
 import re
 import lxml.html
-from billy.scrape.committees import CommitteeScraper, Committee
+from pupa.scrape import Scraper, Organization
 from openstates.utils import LXMLMixin
 
 
-class DECommitteeScraper(CommitteeScraper, LXMLMixin):
+class DECommitteeScraper(Scraper, LXMLMixin):
     jurisdiction = "de"
 
-    def scrape(self, chamber, term):
+    def scrape(self, chamber=None):
+        if chamber:
```

Comment:
This would be more concise using the `chambers = [chamber] if chamber else ['upper', 'lower']` from earlier in this PR.